ui/desktop/src/components/schedule/ScheduleDetailView.tsx

```diff
@@ -21,6 +21,15 @@ import { Loader2, Pause, Play, Edit, Square, Eye } from 'lucide-react';
 import cronstrue from 'cronstrue';
 import { formatToLocalDateWithTimezone } from '../../utils/date';
 import { getSession, Session } from '../../api';
+import {
```

Comment:
with recipes and schedule tracking - do we want to track it this fine grained vs just seeing if people use either feature in some meaningful way? (maybe need to do this but seems like a lot!)